wp-content/themes/core/assets/css/src/theme/forms/_mixins.pcss

```diff
@@ -111,20 +111,20 @@
 
 		display: block;
 		transition: var(--transition);
-		content: var(--form-control-select-ui-icon);
+		content: '\e900';
```

Comment:
I think we should still use an icon var value here (`--icon-chevron-down`), since the actual character code may change depending on the structure of the icon font.

Reply:
👍 have not done a vars pass yet but will add the appropriate chevron var there. For some reason, we had that as the value for `--form-control-select-ui-icon` before I removed it.